docs/sources/get-started/quick-start.md

```diff
@@ -2,160 +2,248 @@
 title: Quickstart to run Loki locally
 menuTitle: Loki quickstart
 weight: 200
-description: How to create and use a simple local Loki cluster for testing and evaluation purposes.
+description: How to create and use a local Loki cluster for testing and evaluation purposes.
+killercoda:
+  title: Loki Quickstart Demo
+  description: This sandbox provides an online enviroment for testing the Loki quickstart demo.
+  details:
+    finish:
+      text: finished.md
```

Comment:
Interested to learn more on what else can go into this details section. Do we add the finished.md markdown here so it's an extra that can be added directly to the killercoda repo?

Reply:
The metadata in the front matter is the Killercoda `index.json` directly, so anything you would put in there, we can put here instead.